R/wbt.R

```diff
@@ -77,24 +173,34 @@ wbt_options <- function(exe_path = NULL, verbose = NULL) {
 #' wbt_verbose(verbose = TRUE)
 #' }
 wbt_verbose <- function(verbose = NULL) {
+  # NA is treated NULL (no effect)
+  if (length(verbose) != 1 || is.na(verbose)) {
+    verbose <- NULL
+  }
   
-  # system environment var takes precedence, but defaults FALSE
-  sysverbose <- Sys.getenv("R_WHITEBOX_VERBOSE", unset = FALSE)
-  if (sysverbose) {
+  # system environment var takes precedence
+  sysverbose <- Sys.getenv("R_WHITEBOX_VERBOSE")
```

Comment:
1. lacks support for `"all"` via system option
2. revisit logic around NA/logical conversion of character `sysverbose`

Reply:
9feb179 gets the behavior I wanted for this option

``` r
library(whitebox)

Sys.setenv("R_WHITEBOX_VERBOSE" = TRUE)
print(wbt_verbose(FALSE))
#> [1] TRUE

Sys.setenv("R_WHITEBOX_VERBOSE" = "all")
print(wbt_verbose(FALSE))
#> [1] TRUE

Sys.setenv("R_WHITEBOX_VERBOSE" = FALSE)
print(wbt_verbose(TRUE))
#> [1] FALSE

Sys.setenv("R_WHITEBOX_VERBOSE" = 0)
print(wbt_verbose(TRUE))
#> [1] FALSE

Sys.setenv("R_WHITEBOX_VERBOSE" = 1)
print(wbt_verbose(FALSE))
#> [1] TRUE
```